spicy/toolchain/bin/spicy-driver.cc

```diff
@@ -67,6 +49,25 @@ class SpicyDriver : public spicy::Driver, public spicy::rt::Driver {
     std::string opt_file = "/dev/stdin";
     std::string opt_parser;
 
+    void fatalError(const std::string& msg) {
+        hilti::logger().error(msg);
+        finishRuntime();
+        spicy::rt::done();
+        hilti::rt::done();
+        exit(1);
+    }
+
+    void fatalError(const hilti::result::Error& error) {
+        hilti::logger().error(error.description());
+
+        if ( error.context().size() )
+            hilti::logger().error(error.context());
+
+        spicy::rt::done();
+        finishRuntime();
```

Comment:
The two `fatalErrors()` aren't consistent: only the 1st calls `hilti::rt::done();` and the order of calling `spicy::rt::done();` and `finishRuntime();` is swapped. Probably all doesn't matter in practice, but still suggest to unitfy to avoid surprises.

Reply:
I missed the first one. Since `finishRuntime` comes from `hilti::rt` it already calls `hilti::rt::done` on its own, so I dropped that in the second call (will now also fix it in the first one). As for ordering, the intention was to do the reverse of the init calls (hilti init'd first, and done'd last).